Libraries/LibCore/EventLoopImplementationWindows.cpp

```diff
@@ -101,17 +101,17 @@ size_t EventLoopImplementationWindows::pump(PumpMode)
     // TODO: investigate if event_count can realistically exceed 64
     VERIFY(event_count <= MAXIMUM_WAIT_OBJECTS);
 
-    HANDLE event_handles[event_count];
-    event_handles[0] = thread_data.wake_event.handle;
+    AK::Vector<HANDLE> event_handles;
```

Comment:
Nit: 
No need for the `AK::` prefix. You might also want to consider setting an inline size to that vector: `Vector<HANDLE, MAXIMUM_WAIT_OBJECTS>` or whatever value seems appropriate.